lib/robots/dor_repo/gis_delivery/start_accession_workflow.rb

```diff
@@ -10,8 +10,7 @@ def initialize
 
         def perform_work
           logger.debug "start-accession-workflow working on #{bare_druid}"
-          current_version = object_client.version.current
-          workflow_service.create_workflow_by_name(druid, 'accessionWF', version: current_version)
+          object_client.version.close(description: 'Closed by GIS delivery robot to initiate accessioning')
```

Comment:
No need to provide a description.

This is a description of the version (not the closing operation). It can optionally be provided when closing, but isn't necessary.